metadata/metadata.go

```diff
@@ -90,7 +90,7 @@ func readFile(file string) ([]byte, error) {
 	return ioutil.ReadFile(file)
 }
 
-const RKEVersionDev = "v0.2.3"
+const RKEVersionDev = "v1.1.1-rc100"
```

Comment:
Why are we changing this?

Reply:
This is required as the minimum RKE version for 1.18.2 has been set to v1.1.1-rc99:

https://github.com/aiyengar2/kontainer-driver-metadata/blob/afe5b43a12112a86eccee8281f89fd4d2e50ae4e/rke/k8s_version_info.go#L120-L123